client/my-sites/site-settings/composing/default-post-format.jsx

```diff
@@ -32,7 +32,7 @@ const DefaultPostFormat = ( {
 		<FormFieldset>
 			{ siteId && <QueryPostFormats siteId={ siteId } /> }
 
-			<FormLabel htmlFor="default_post_format">{ translate( 'Default Post Format' ) }</FormLabel>
+			<FormLabel htmlFor="default_post_format">{ translate( 'Default post format' ) }</FormLabel>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **110** times:
`translate( 'Default post format.' )` *ES Score: 8*